cylc/flow/scripts/edit.py

```diff
@@ -102,8 +106,8 @@ def get_option_parser():
 
 
 @cli_function(get_option_parser)
-def main(parser, options, *args):
-    flow_file = parse_workflow_arg(options, args[0])[1]
+def main(parser: COP, options: 'Values', reg: str) -> None:
+    flow_file = str(parse_reg(reg, src=True)[1])
```

Comment:
Not 100% sure but the variable was previously a string, so I think I did this to be safe in case something further down doesn't like using pathlib Paths instead of strings